mscore/mixer.cpp

```diff
@@ -238,16 +238,53 @@ void Mixer::retranslate(bool firstTime)
 
 void Mixer::closeEvent(QCloseEvent* ev)
       {
+      std::cout<<"Mixer closeEvent"<<endl;
```

Comment:
You could use `qDebug()` for all these

Reply:
Ooh. I'll look that up. std:cout etc. felt very clunk. Thanks. Just testing the mixer solution - encouraging so far - and if it still looks good will push up shortly.